geometry_msgs/PoseArray[] segments
